### ISSUE TITLE
Clickup Trigger Test fails

### ISSUE THREAD

**Issue by obuchmann** [n8n-io/n8n#8581]
**Describe the bug**
When you use a Clickup Trigger based on an event. E.g. "task.updated" the Trigger fails instant on receiving the event with the error:
ERROR: The "key" argument must be of type string or an instance of ArrayBuffer, Buffer, TypedArray, DataView, KeyObject, or CryptoKey. Received undefined

**To Reproduce**

1. Create a new Workflow
2. Add Clickup Trigger "On Task Updated"
3. Click "Test step"
4. Update a Task in Clickup
5. See error

**Expected behavior**
The Trigger should not fail and provide the event information.

**Environment (please complete the following information):**

- OS: Ubuntu Linux 22.04 (Latest Docker)
- n8n Version 1.26.0 (Latest official Docker Image)
- Node.js Version default Docker Image Version
- Database system SQLite
- Operation mode default

**Additional context**
```
Exception: 
TypeError: The "key" argument must be of type string or an instance of ArrayBuffer, Buffer, TypedArray, DataView, KeyObject, or CryptoKey. Received undefined
    at new NodeError (node:internal/errors:405:5)
    at prepareSecretKey (node:internal/crypto/keys:596:11)
    at new Hmac (node:internal/crypto/hash:134:9)
    at createHmac (node:crypto:165:10)
    at Object.webhook (/usr/local/lib/node_modules/n8n/node_modules/n8n-nodes-base/dist/nodes/ClickUp/ClickUpTrigger.node.js:319:59)
    at Workflow.runWebhook (/usr/local/lib/node_modules/n8n/node_modules/n8n-workflow/dist/Workflow.js:641:38)
    at Object.executeWebhook (/usr/local/lib/node_modules/n8n/dist/WebhookHelpers.js:213:48)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at /usr/local/lib/node_modules/n8n/dist/TestWebhooks.js:97:37
```

**Comment by Joffcom** [comment]
Hey @obuchmann 

Can you try using 1.27.2 or 1.28.0  and let me know if the issue is resolved, it should be fixed from 1.27.0.

If you are still seeing the issue after updating let me know.

**Comment by obuchmann** [comment]
@Joffcom I can confirm this is fixed in 1.27.2, thanks!
